tests/container/BUILD

```diff
@@ -1,120 +0,0 @@
-# Copyright 2017 The Bazel Authors. All rights reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#    http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-licenses(["notice"])  # Apache 2.0
-
-package(default_visibility = ["//visibility:public"])
-
-load("@base_images_docker//package_managers:download_pkgs.bzl", "download_pkgs")
-load("@base_images_docker//package_managers:install_pkgs.bzl", "install_pkgs")
-load("@base_images_docker//package_managers:apt_key.bzl", "add_apt_key")
-load("//rules/container:docker_toolchains.bzl", "toolchain_container")
-
-# This file contains some sample targets that excersise the container rules
```

Comment:
lets leave some of these basic tests for toolchain_container

Reply:
Maybe the one that we want to leave is the one currently in tests/config/BUILD (so that it can be used to test toolchain_container and then also test docker_toolchain_autoconfig), I do think if so, we should move the toolchain_container target from that build file (https://github.com/bazelbuild/bazel-toolchains/blob/master/tests/config/BUILD#L77 ) to this one

Reply:
done